packages/node_modules/@node-red/nodes/core/logic/10-switch.html

```diff
@@ -48,6 +48,7 @@
         {v:"lte",t:"<=",kind:'V'},
         {v:"gt",t:">",kind:'V'},
         {v:"gte",t:">=",kind:'V'},
+        {v:"exists",t:"switch.rules.exists",kind:'V'},
```

Comment:
Can we call this rule `has` internally rather than `exists`? May as well start consistent (plus is a couple chars shorter 😉) and I can imagine `exists` being the name of a completely different rule.

Reply:
apart from when you use the length to set the input box...

     if (type.length < 4) {
           selectWidth = 60;
making it "has" means the actual label won't fit.

Reply:
it's only internal so how about hask ?